docs/reference/objects/product/index.md

```diff
@@ -47,20 +51,15 @@ string
 
 The name of the country the product is based in.
 
-## `product.dates`
```

Comment:
is `product.dates` depreciated? If so please can it be kept in and marked depreciated instead?